hack/tools/Makefile

```diff
@@ -116,6 +117,11 @@ kubectl: $(KUBECTL) ## Install kubectl
 $(ETCD) $(KUBE_APISERVER) $(KUBECTL): k8s-envtest  ## Install envtest related binaries
 	@cp -f "$(ENVTEST_DOWNLOAD_DIR)/$(@F)" $(BIN_DIR)
 
+.PHONY: $(GOVULNCHECK)
```

Comment:
This does not need to be phony. The binary will be rebuilt if `go.mod` changes.

Reply:
I spoke with @aruneshpa offline. He made this `PHONY` since the other targets were phony as well. We're going to leae this `PHONY` for now and clean up all the targets in a separate PR that do not need to be `PHONY`.